llvm/unittests/IR/IntrinsicsTest.cpp

```diff
@@ -106,4 +107,67 @@ TEST_F(IntrinsicsTest, InstrProfInheritance) {
     EXPECT_TRUE(Checker(*Intr));
   }
 }
+
+TEST(IntrinsicVerifierTest, LRound) {
```

Comment:
Would be easier to just write regular IR in test/Assembler

Reply:
I am planning to extend the same to lrint as well.

Reply:
Same? Can just write an assembler test?